xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-api/src/main/java/org/xwiki/security/authorization/DefaultAuthorizationManager.java

```diff
@@ -45,6 +45,8 @@
 @Singleton
 public class DefaultAuthorizationManager implements AuthorizationManager
 {
+    private static final String GROUPS_PREFIX = "xwiki:Groups";
```

Comment:
I don't think this would work on a multi-wiki instance (since the part before the colon would be different), but we never use that anyway, so I'm not concerned.